core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -899,6 +924,7 @@ package object config {
         "driver and executor environments contain sensitive information. When this regex matches " +
         "a property key or value, the value is redacted from the environment UI and various logs " +
         "like YARN and event logs.")
+      .version("2.1.2")
```

Comment:
SPARK-18535 and SPARK-19720, commit ID: 444cca14d7ac8c5ab5d7e9d080b11f4d6babe3bf#diff-6bdad48cfc34314e89599655442ff210